components/bl-accordion-component/README.md

```diff
@@ -0,0 +1,58 @@
+# Accordion
+
+Accordion is a component of Backendless UI-Builder designer. It allows the user to show and hide sections of related content on a page. The component supports basic and controlled type of accordion.
+
+Customizations include adjustments of font size, color, background color and padding for both title and content.
+
+## Usage
+
+After adding the component to the page, specify the accordion data. This can be done in UI-Builder designer or using codeless logic.
+
+### Styles and Settings
+
+<dl>
+<dt>Accordion Data</dt>
+<dd>Array of objects that define the title and content of the accordion sections. Can be determined by Accordion Data Logic.</dd>
```

Comment:
Could we add a signature for that objects?

Reply:
Could you please explain what you mean? 
The default value of the accordion data shows the user how objects should look or that's not what you meant?

Reply:
> The default value of the accordion data shows the user how objects should look or that's not what you meant?

Yes, but that must be documented too. So add a signature at descriptions:

[{title: <string>, content: <string>}]